src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -76,7 +76,8 @@ private static String fireAt(Request req, Response res) {
             res.status(400);
             return result;
         } else //(result == null)
-            currModel.smartShootAtPlayer();
+            //currModel.smartShootAtPlayer();
```

Comment:
This comment should be deleted or changed. It's actually a little confusing, as you've deleted currModel.smartShootAtPlayer(). If you wanted to comment on what that line of code used to be, maybe add that in. For example, "This used to be currModel.smartShootAtPlayer()" .